### ISSUE TITLE
Tab Index in the main nav

### ISSUE THREAD

**Issue by nathanyoung** [CenturyLinkCloud/Cyclops#65]
The main nav should handle keyboard navigation better (tab/shift-tab). When a flyout is expanded, the top order should be updated so that the first item in the flyout has focus. Shift-tab should put focus back on the top level main-nav.
